packages/design-system/src/icons/svgs/core-24/create-file-24.icon.ts

```diff
@@ -0,0 +1,24 @@
+import {toniqIconColors} from '../../../styles/icon-colors';
+import {html, ToniqSvg} from '../../toniq-svg';
+
+export const CreateFile24Icon = new ToniqSvg(
+    'CreateFile24Icon',
+    html`
+        <svg
```

Comment:
This SVG needs to be optimized and likely redrawn. I'll get that to you within the hour.

Reply:
Here you go: 

```

        <svg
            xmlns="http://www.w3.org/2000/svg"
            xml:space="preserve"
            fill-rule="evenodd"
            stroke-linecap="round"
            stroke-linejoin="round"
            clip-rule="evenodd"
            stoke-miterlimit="1.5"
            viewBox="0 0 24 24"
            fill="none"
            stroke-width="2px"
            stroke="${toniqIconColors.stroke()}"
            width="24"
            height="24"
        >
            <path
                d="M11 15V7m-4 4h8m4-6.4c0-.9-.7-1.6-1.6-1.6H4.6C3.7 3 3 3.7 3 4.6v12.8c0 .9.7 1.6 1.6 1.6h12.8c.9 0 1.6-.7 1.6-1.6V4.6ZM6 19v1.4c0 .9.7 1.6 1.6 1.6h12.8c.9 0 1.6-.7 1.6-1.6V7.6c0-.9-.7-1.6-1.6-1.6H19"
            />
        </svg>
```